control/getCategories.js

```diff
@@ -0,0 +1,12 @@
+// const model = require('seed/lib/seed/errors/model')
+const database = require('../models/getdb.js')
```

Comment:
Our database connection (where we are exporting a new pool) should be set up already and coming from connection.js.
Our models folder should only have our model functions inside (i.e. any functions where we are interacting with our database)

Reply:
VSCode put line 1 in automatically and I just commented it out as opposed to deleting it just in case it was needed.